great-idea/css/index.css

```diff
@@ -63,4 +63,333 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* START OF CUSTOM CSS */
+
+body {
+	width: 900px;
+	margin: auto;
+	text-align: center;
+	background-color: rgb(14, 14, 20);
+	color: rgb(201, 200, 200);
+}
+
+p {
+	margin: 20px 20px 0 0;
+}
+
+hr {
+	border: 1px solid white;
+}
+
+h3 {
+	font-family: 'Open-Sans', sans-serif;
+	font-weight: 600;
+	color: white;
+}
+
+h1 {
+	color: white;
+}
+
+/* Nav Section */
+
+nav {
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+	position: fixed;
+	top: 0;
+	padding: 10px 0;
+	text-align: left;
+	background-image: linear-gradient(to right, rgb(38, 38, 68) , rgb(90, 90, 161));
```

Comment:
I'm glad you found the linear gradient useful... 
I have to say I absolutely love the color selections throughout this entire product.
Great stuff!